### PR TITLE
adds hashicorp packer.io module for automated machine image builds

### PR DESCRIPTION
**Pull request by xinau**
##### SUMMARY
added hashicorp packer.io module for automated machine image builds

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
hashicorp packer module used for machine image building and directly provisioning servers with said image

##### ANSIBLE VERSION
```console
$ ansible --version
ansible 2.2.1.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
```

##### ADDITIONAL INFORMATION
```yaml
- name: "updates image with packer"
  hosts: 127.0.0.1
  connection: local
  tasks:
    - name: "building images"
      packer: 
        path: "./image.json"
        chdir: "{{ playbook_dir}}/files/packer"
        vars:
          image_name: "test-1234"
```

### CODE REVIEW

**Comment by xinau** [comment]
@jborean93  or @mattclay  could one of you tell me how to fix this error ?
```
Run command: /ansible/test/runner/.tox/py27/bin/python test/sanity/validate-modules/validate-modules --format json --arg-spec lib/ansible/modules/cloud/misc/packer.py
ERROR: Found 1 validate-modules issue(s) which need to be resolved:
ERROR: lib/ansible/modules/cloud/misc/packer.py:0:0: E316 ANSIBLE_METADATA.metadata_version: not a valid value for dictionary value @ data['metadata_version']. Got '1.0'
```

File: lib/ansible/modules/cloud/misc/packer.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Felix Ehrenpfort <felix.ehrenpfort@codecentric.cloud>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '0.1',
```
Comment (mattclay) [inline lib/ansible/modules/cloud/misc/packer.py]: `metadata_version` should be `1.1` instead of `0.1`.

Comment (xinau) [reply lib/ansible/modules/cloud/misc/packer.py]: thanks, changed it.
now it works! :)

**Comment by xinau** [comment]
@jborean93, @mattclay how can i get the PR merged?

**Comment by luckylittle** [comment]
@xinau You cannot do it on your own obviously, the maintainers of the similar modules will have to review it first and tag it. Good work you did there!

**Pull request closed without merging**